src/main/Tree.csproj

```diff
@@ -37,5 +36,9 @@
 	<ItemGroup>
 		<EmbeddedResource Include="wwwroot\**\*" />
 	</ItemGroup>
+
```

Comment:
Blazor.Common should be referenced as a package reference.

Reply:
I make it reference as a project because I make a changes to Blazor.common project